baseline.yaml

```
@@ -649,6 +649,30 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-15
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project documentation MUST include a 
+       descriptive statement about how the project
```

```suggestion
       describe how the project
```

A "statement" often implies something short, that is, a sentence or "single declaration or remark". We should not limit them to only give one sentence, and this change makes the criterion text shorter too.